### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by psonnera** [tremor/nightscout-at-oraclevm#1]
Hello, do you use creationix/nvm only to get nodejs installed?

**Comment by tremor** [comment]
Yes, because I want to install this known correct working specific version and don't want to automatically upgrade.
